packages/hop-node/src/watchers/HealthCheckWatcher.ts

```diff
@@ -797,4 +841,75 @@ export class HealthCheckWatcher {
       }
     })
   }
+
+  async getUnrelayedTransfers (): Promise<UnrelayedTransfers[]> {
+    const now = DateTime.now().toUTC()
+    const endDate = now.minus({ hours: 1 })
+    const startDate = endDate.minus({ days: this.days })
+    const tokens = ''
```

Comment:
same comment as above about making the token parameter be the last optional parameter